### PR TITLE
Update the "getting started" guide to Rails 6.1

### PR DESCRIPTION
**Pull request by oliverklee**
This hopefully will reduce confusion for developers who use the
current Rails version.

### CODE REVIEW

**Comment by p8** [comment]
Hi @oliverklee, thanks for this PR.
I think there is another migration that needs to be updated:
```ruby
class CreateComments < ActiveRecord::Migration[6.0]
```
The following line could be updated as well:
```
If it says something like "Rails 6.0.0", you are ready to continue.
```

For changes to documentation you can add `[ci skip]` to the commit message to skip running all the tests.

**Comment by oliverklee** [comment]
@p8 Thanks! Will change both.

**Comment by oliverklee** [comment]
Changed accordingly and repushed.

File: guides/source/getting_started.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ run the following:
 $ rails --version
 ```
 
-If it says something like "Rails 6.0.0", you are ready to continue.
+If it says something like "Rails Rails 6.1.3.1", you are ready to continue.
```
Comment (zzak) [inline guides/source/getting_started.md:163]: I'm reading this as, could you fix it?

> "Rails Rails 6.1.3.1"

Also, I'm unsure the value in updating this version every time a new release to Rails is made, perhaps a rough rule would be "every minor version" so 6.1.0 would be sufficient here.

Comment (oliverklee) [reply guides/source/getting_started.md:163]: Oh, good catch! Will change.

Comment (oliverklee) [reply guides/source/getting_started.md]: Fixed, rebased and repushed.

**Review by zzak — APPROVED** [review]

**Pull request merged**